scripts/build_go_slim.sh

```diff
@@ -48,6 +53,13 @@ done
 
 # Check dependencies
 dependencies go
+if [[ -n $compress ]]; then
+	dependencies tar zstd
+
+	if [[ ! $compress == [0-9]* ]] || ((compress > 22)) || ((compress < 1)); then
```

Comment:
```suggestion
	if [[ "$compress" != [0-9]* ]] || [ "$compress" -gt 22 ] || [ "$compress" -lt 1 ]; then
```

Reply:
Just curious why we'd want to avoid `(( ))`? Neither handles non-numbers gracefully (but we've verified that with the glob match).

Reply:
Feel free to change it across all scripts 👍

Reply:
I won't do it in this PR to keep the noise down, but I can do a follow-up tomorrow.